languages/ruby/exercises/concept/booleans/.meta/design.md

```diff
@@ -0,0 +1,49 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of "nil" in Ruby.
+
+## Learning objectives
+
+- Know that `true` is a Ruby object used to represent logical `true`
+- Know that everything except `false` and `nil` is truthy
+- Know that `false` is a Ruby object used to represent logical `false`
+- Know that `false` is falsey
+- Know that `nil` is a Ruby object used to represent an "empty" or "default" value
```

Comment:
```suggestion
- Know that `nil` is a Ruby object used to represent "nothingness"
```

I think that it will be noticed that it is often used as a default for various things.  We can talk about it when instantiating those things (such as creating `Hash` where it happens that `nil` is returned) later.